static/panes/editor.js

```diff
@@ -500,6 +519,55 @@ Editor.prototype.updateOpenInQuickBench = function () {
         text: this.getSource()
     };
 
+    var compilers = this.getCompilerStates();
+
+    _.each(compilers, function (compiler) {
+        var knownCompiler = false;
+        if (compiler.compiler.match(/^g[0-9]*$/)) {
+            knownCompiler = true;
+            quickBenchState.compiler = "gcc-" + compiler.compiler[1] + "." + compiler.compiler[2];
```

Comment:
these will break for gcc/clang 10

maybe we can add something in the server side properties?
```
compiler.g92.quickBenchCompiler=gcc-9.2
```

we could then set defaults for older gcc/clang versions:
```
group.gcc86.quickBenchCompiler=gcc-5.5
```

Reply:
hmm, didn't think of that.

Reply:
Thanks both: good catch @apmorton but great start @partouf

Reply:
Changed to use semver instead, felt uncomfortable with adding a special property for it. Though we can always still do that if this gives us too much trouble.